features/homepage-carousel.feature

```diff
@@ -1,26 +0,0 @@
-@homepage
-Feature: Homepage carousel
-
-  Rules:
-  - Contains between 1 and 3 covers
-  - Any type eLife content can have a cover
-  - A cover has a title and image, which can be different from the content that it links to
-
-  Background:
-    Given 10 articles have been published
-
-  @javascript
-  Scenario: Covers use details from the content that they link to by default
-    Given there is a collection called 'Tropical disease'
-    And there is a cover linking to the 'Tropical disease' collection
```

Comment:
These 2 steps were important that there is a cover item available. Without these or something similar to replace then there is no response when the `/covers/current` api call is made when accessing the home page.